src/main/scala/higherkindness/rules_scala/common/worker/WorkerMain.scala

```diff
@@ -59,13 +59,14 @@ trait WorkerMain[S] {
                 1
             }
 
+            out.flush()
```

Comment:
Why do you move this line?

Reply:
The contract of `PrintStream` says that it might keep some content buffered until this is called. Thus, `outStream.toString` might not contain some of the output sent to `out`, which would be dropped. In practice I didn't see this but according to the contract, it's possible. An alternative might be to set `autoFlush` to true, but that still might miss content if a partial line is written.

Reply:
It seems to be unrelated to log level. I prefer to undo this change. We can have another PR to discuss this problem.